src/scripts/main.js

```diff
@@ -1,8 +1,29 @@
 'use strict';
 
 const people = require('./lib/people');
+const table = document.querySelector('.dashboard');
 
-// eslint-disable-next-line no-console
-console.log(people); // you can remove it
+people.forEach(person => {
+  const tableRow = document.createElement('tr');
 
-// write your code here
+  const age = person.died - person.born;
+  const century = Math.ceil(person.died / 100);
+
+  const personArray = [
+    person.name,
+    person.sex,
```

Comment:
Gender should be set in the correct format (Male/Female)
![image](https://github.com/mate-academy/js_task_generate_table_DOM/assets/96409563/bec833a3-6bef-49d4-a706-30661d5cbe62)